src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1750,19 +1750,26 @@ private IVariableDeclarationGroupOperation CreateBoundMultipleLocalDeclarationsB
             bool declarationIsImplicit = boundMultipleLocalDeclarations.WasCompilerGenerated;
             IVariableDeclarationOperation multiVariableDeclaration = new CSharpLazyVariableDeclarationOperation(this, boundMultipleLocalDeclarations, _semanticModel, declarationSyntax, null, default, declarationIsImplicit);
 
-            // If this is a using declaration, work out the declaration kind
-            VariableDeclarationKind declKind = boundMultipleLocalDeclarations is BoundUsingLocalDeclarations usingDecl
-                                               ? usingDecl.AwaitOpt is object
-                                                    ? VariableDeclarationKind.AsynchronousUsing
-                                                    : VariableDeclarationKind.Using
-                                               : VariableDeclarationKind.Default;
-
             ITypeSymbol type = null;
             Optional<object> constantValue = default(Optional<object>);
             // If the syntax was the same, we're in a fixed statement or using statement. We make the Group operation implicit in this scenario, as the
             // syntax itself is a VariableDeclaration
             bool isImplicit = declarationGroupSyntax == declarationSyntax || boundMultipleLocalDeclarations.WasCompilerGenerated;
-            return new VariableDeclarationGroupOperation(ImmutableArray.Create(multiVariableDeclaration), declKind, _semanticModel, declarationGroupSyntax, type, constantValue, isImplicit);
+            var variableDeclaration = new VariableDeclarationGroupOperation(ImmutableArray.Create(multiVariableDeclaration), _semanticModel, declarationGroupSyntax, type, constantValue, isImplicit);
+
+            if (boundMultipleLocalDeclarations is BoundUsingLocalDeclarations usingDecl)
+            {
+                return new UsingDeclarationOperation(
+                    variableDeclaration,
+                    isAsynchronous: usingDecl.AwaitOpt is object,
+                    _semanticModel,
+                    declarationGroupSyntax,
+                    type: null,
+                    constantValue: default,
+                    isImplicit: true);
```

Comment:
Should this be the other way around? So that the declarationGroup is implicit and the UsingDecl is not? That's how it works for using/fixed statements

Reply:
I think that might break existing clients who look for explicitly declared variable declarations?